doc/patch-author-guide.md

```diff
@@ -365,13 +377,26 @@ Some examples:
   function will also change.  In this case there's nothing you can do to
   prevent the extra changes.
 
+* If a changed function was originally inlined, but turned into a callable
+  function after patching, consider adding `inline` to the function
+  definition.  Likewise, if a function is only inlined after patching,
+  consider `noinline`.  Remember though, these keywords are only suggestions
+  to the compiler.  YMMV.
```

Comment:
I've never actually needed to use these before, thanks for reminding me about this.